### PR TITLE
Mq changes

### PR DESCRIPTION
**Pull request by jbygdell**
* Updates related to MQ changes
* Mapper is added to sda-svc chart

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -479,13 +479,13 @@ jobs:
...
-        broker.exchange="lega",\
-        broker.vhost="lega",\
-        broker.queue.inbox="files.inbox",\
-        broker.queue.completed="files.completed",\
-        broker.queue.verify="lega",\
+        broker.exchange="sda",\
+        broker.vhost="sda",\
+        broker.queue.inbox="inbox",\
+        broker.queue.completed="completed",\
+        broker.queue.verify="archive",\
```
Comment (pontus) [inline .github/workflows/deploy.yml:486]: Felskrivet? `archive` finns inte så vitt jag kan se. Hårdkodat till `archived` i `verify-deploy.yaml` om jag läser rätt.

**Review by pontus — APPROVED** [review]

File: sda-svc/templates/sftp-inbox-deploy.yaml [hunk 2]
```diff
@@ -71,6 +71,8 @@ spec:
               key: mqPassword
         - name: BROKER_PORT
           value: {{ .Values.global.broker.port | quote }}
+        - name: BROKER_ROUTINGKEY
+          value: "inbox"
```
Comment (blankdots) [inline sda-svc/templates/sftp-inbox-deploy.yaml:75]: shouldn't this come from `global.inbox.brokerRoutingKey`

File: sda-svc/templates/sftp-inbox-deploy.yaml [hunk 3]
```diff
@@ -71,6 +71,8 @@ spec:
               key: mqPassword
         - name: BROKER_PORT
           value: {{ .Values.global.broker.port | quote }}
```
Comment (blankdots) [inline sda-svc/templates/sftp-inbox-deploy.yaml:73]: i think we should add  a var for broker exchange, to be consistent with rest of services and we should add it to `values.yaml`
```
- name: BROKER_EXCHANGE
  value: {{ default "sda" .Values.inbox.brokerExchange }}
```

File: sda-svc/README.md [hunk 4]
```diff
@@ -87,7 +87,7 @@ Parameter | Description | Default
 `global.elixir.pubKey` | Public key used to verify Elixir JWT. | `""`
 `global.elixir.jwkPath` | Path on the oicd Host where the JWK definitions can be found. | `/oidc/jwk`
 `global.elixir.oidcdHost` | URL to get the public key used to verify Elixir JWT. | `"https://login.elixir-czech.org"`
-`global.inbox.brokerRoutingKey` | Routing key the inbox uses when publishing messages. | `files.inbox"`
+`global.inbox.brokerRoutingKey` | Routing key the inbox uses when publishing messages. | `inbox"`
```
Comment (blankdots) [inline sda-svc/README.md:90]: should this be also in `values.yaml` ?

also adding `global.inbox.brokerExchange` with default to `sda` ?

**Review by blankdots — APPROVED** [review]
Small improvements suggested, otherwise approved.

also small config concern in mapper, as how it seems now it requires a routkingkey but does not use it. How I read this https://github.com/neicnordic/sda-pipeline/blob/master/internal/config/config.go#L83-L85 is that mapper needs the `broker.routingkey` even though it does not use it.

File: sda-svc/templates/sftp-inbox-deploy.yaml (hunk 2 above)
Comment (jbygdell) [reply sda-svc/templates/sftp-inbox-deploy.yaml:77]: That should be fixed in sda-pipeline.

**Comment by jbygdell** [comment]
> Small improvements suggested, otherwise approved.
> 
> also small config concern in mapper, as how it seems now it requires a routkingkey but does not use it. How I read this https://github.com/neicnordic/sda-pipeline/blob/master/internal/config/config.go#L83-L85 is that mapper needs the `broker.routingkey` even though it does not use it.

That should be fixed in sda-pipeline

File: sda-svc/templates/sftp-inbox-deploy.yaml (hunk 3 above)
Comment (jbygdell) [reply sda-svc/templates/sftp-inbox-deploy.yaml:73]: `inbox.brokerExchange` is obsolete as all services talk to the same exchange now.

Comment (jbygdell) [reply sda-svc/templates/sftp-inbox-deploy.yaml:73]: But it should be fixed in the s3-proxy code.

**Review by blankdots — APPROVED** [review]

**Pull request merged**